src/k8s-extension/azext_k8s_extension/partner_extensions/OpenServiceMesh.py

```diff
@@ -8,13 +8,22 @@
 from azure.cli.core.azclierror import InvalidArgumentValueError, RequiredArgumentMissingError
 from knack.log import get_logger
 
-from ..vendored_sdks.models import ExtensionInstance
-from ..vendored_sdks.models import ExtensionInstanceUpdate
-from ..vendored_sdks.models import ScopeCluster
-from ..vendored_sdks.models import Scope
+from msrestazure.azure_exceptions import CloudError
+
+from azure.cli.core.commands.client_factory import get_subscription_id
+
+from pyhelm.chartbuilder import ChartBuilder
+from packaging import version
+import yaml
+
+from azext_k8s_extension.partner_extensions.PartnerExtensionModel import PartnerExtensionModel

```

Comment:
You are importing the same package twice, also we require relative importing to build both the k8s-extension and the k8s-extension-private packages